include/z64.h

```diff
@@ -869,15 +807,14 @@ typedef struct {
     /* 0x10 */ OSTime resetTime;
 } PreNmiBuff; // size = 0x18 (actually osAppNmiBuffer is 0x40 bytes large but the rest is unused)
 
-typedef struct PlayState PlayState;
 
-typedef s32 (*ColChkResetFunc)(PlayState*, Collider*);
-typedef void (*ColChkBloodFunc)(PlayState*, Collider*, Vec3f*);
-typedef void (*ColChkApplyFunc)(PlayState*, CollisionCheckContext*, Collider*);
-typedef void (*ColChkVsFunc)(PlayState*, CollisionCheckContext*, Collider*, Collider*);
-typedef s32 (*ColChkLineFunc)(PlayState*, CollisionCheckContext*, Collider*, Vec3f*, Vec3f*);
+typedef s32 (*ColChkResetFunc)(struct PlayState*, Collider*);
+typedef void (*ColChkBloodFunc)(struct PlayState*, Collider*, Vec3f*);
+typedef void (*ColChkApplyFunc)(struct PlayState*, CollisionCheckContext*, Collider*);
+typedef void (*ColChkVsFunc)(struct PlayState*, CollisionCheckContext*, Collider*, Collider*);
+typedef s32 (*ColChkLineFunc)(struct PlayState*, CollisionCheckContext*, Collider*, Vec3f*, Vec3f*);
 
-typedef void(*room_draw_func)(PlayState* play, Room* room, u32 flags);
+typedef void(*RoomDrawFunc)(struct PlayState* play, Room* room, u32 flags);
```

Comment:
This is changed and addressed in https://github.com/zeldaret/mm/pull/1143, so I'd leave this alone to prevent conflicts

Reply:
Did what I could. Note this line will still conflict since I needed to leave the `struct Playstate` change.